inst/pages/Introductory-workflow.qmd

```diff
@@ -0,0 +1,353 @@
+# Introductory
```

Comment:
It seems that the chapter starts with "Chapter 2 - Introductory". I guess this should be "## Introductory" (Render the book as check that everything is correct)